docker-compose.yml

```diff
@@ -0,0 +1,87 @@
+version: '2'
+services:
+  web:
+    build:
+      context: ./resources/docker/web
+    ports:
+      - "8080:8080"
+    volumes:
+      - .:/var/www/html
+    environment:
+      XDEBUG_CONFIG: "remote_enable=1 xdebug.remote_connect_back=1 remote_host=10.254.254.254 remote_port=9000 idekey=PHPSTORM remote_autostart=0"
```

Comment:
I think xdebug.remote_connect_back=1 should be remote_connect_back=1

Reply:
This has been fixed.